models.py

```diff
@@ -1132,6 +1132,7 @@ class CognitionProject(Base):
 
     allow_file_upload = Column(Boolean, default=False)
     max_file_size_mb = Column(Float, default=3.0)
+    max_folder_size_mb = Column(Float, default=20.0)
```

Comment:
i think this is not float anymore since we have a dropdown with options

Reply:
It is not relevant for the database, it is still a float there